src/Styleguide/Components/FollowArtistPopover/FollowArtistPopoverRow.tsx

```diff
@@ -55,9 +57,28 @@ class FollowArtistPopoverRow extends React.Component<Props, State> {
         variables: {
           input: { artist_id: artistID, unfollow: false },
         },
+        optimisticUpdater: () => {
```

Comment:
Nice! I've only seen `optimisticResponse` used where you'd include the response with `is_followed: true`, but this actually feels a whole lot better.